CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ### All Packages
 
 * Fix an accidental extra global variable in the UMD builds. ([@gaearon](https://github.com/gaearon) in [#10935](https://github.com/facebook/react/pull/10935))
+* Deprecate use of Bower for accessing React packages. We will no longer be using Bower. ([@bvaughn](https://github.com/bvaughn) in [#11223](https://github.com/facebook/react/pull/11223))
```

Comment:
This sounds like something we've done in React itself. It also sounds like existing Bower users will be negatively affected.

I would instead add a separate section like 

>### Discontinuing Bower Releases
>
>Starting with 16.1.0, we will no longer be publishing new releases on Bower. You can continue using Bower for old releases, or point your Bower configs to [React UMD builds hosted on unpkg](https://reactjs.org/docs/installation.html#using-a-cdn).

Reply:
Good idea! Will fix.